src/lab16.py

```diff
@@ -167,7 +167,7 @@ def is_focusable(node):
 def print_tree(node, indent=0):
     print(' ' * indent, node)
     children = node.children
-    if not isinstance(children, list):
+    if isinstance(children, ProtectedField):
```

Comment:
Uhh, this change doesn't appear in the book and does not change behavior, so we don't have to do it. But lower down `tree_to_list` does `isinstance(children, ProtectedField)` instead of `not isinstance(children, list)`, and there we can't change the code, so changing `print_tree` at least makes both functions do the same thing the same way.